src/lcals/FIRST_MIN-Hip.cpp

```diff
@@ -27,6 +27,8 @@ namespace lcals
 #define FIRST_MIN_DATA_TEARDOWN_HIP \
   deallocHipDeviceData(x);
 
+FIRST_MIN_MINLOC_COMPARE;	
```

Comment:
This line should be removed. It causes a multiply defined symbol linker error.

Reply:
Hmm. Needs to be forward declared somewhere or else we get a hard compile error.

Reply:
You're right. It works with CUDA because we typically enable OpenMP and CUDA, but we haven't been turning on OpenMP with HIP builds. It's forward declared in FIRST_MIN-OMP.cpp. I recently fixed the RAJA CMake stuff to support OpenMP with HIP also.

I think the right thing to do is to forward declare in FIRST_MIN-Cuda.cpp and FIRST_MIN-Hip.cpp like you had it and guard the declarations in those spots with #if !defined(RAJA_ENABLE_OPENMP). Does that make sense?

Reply:
Maybe better to just forward declare once in FIRST_MIN.cpp, which will get compiled in any case.

Reply:
My ideas here don't work for all cases. I will push a change to fix the issue shortly.